acquisitions/src/main/resources/thunderjet/mod-invoice/features/approve-and-pay-invoice-with-past-fiscal-year.feature

```diff
@@ -0,0 +1,104 @@
+# For https://issues.folio.org/browse/MODINVOICE-463
+@parallel=false
+Feature: Approve and pay invoice with past fiscal year
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def createFiscalYear = read('classpath:thunderjet/mod-finance/reusable/createFiscalYear.feature')
+    * def createInvoice = read('classpath:thunderjet/mod-invoice/reusable/create-invoice.feature')
+    * def createInvoiceLine = read('classpath:thunderjet/mod-invoice/reusable/create-invoice-line.feature')
+    * def approveInvoice = read('classpath:thunderjet/mod-invoice/reusable/approve-invoice.feature')
+    * def payInvoice = read('classpath:thunderjet/mod-invoice/reusable/pay-invoice.feature')
+
+    * def pastFiscalYearId = callonce uuid1
+    * def presentFiscalYearId = callonce uuid2
+    * def ledgerId = callonce uuid3
+    * def fundId = callonce uuid4
+    * def budgetId = callonce uuid5
+    * def invoiceId = callonce uuid6
+    * def invoiceLineId = callonce uuid7
+
+
+  Scenario: Create 2 fiscal years, ledger, fund and budget
+    * configure headers = headersAdmin
+    * def v = call createFiscalYear { id: #(pastFiscalYearId), code: 'TESTFY2020', periodStart: '2020-01-01T00:00:00Z', periodEnd: '2020-12-30T23:59:59Z', series: 'TESTFY' }
+    * def currentYear = call getCurrentYear
+    * def currentStart = currentYear + '-01-01T00:00:00Z'
+    * def currentEnd = currentYear + '-12-30T23:59:59Z'
+    * def v = call createFiscalYear { id: #(presentFiscalYearId), code: #('TESTFY' + currentYear), periodStart: #(currentStart), periodEnd: #(currentEnd), series: 'TESTFY' }
+    * def v = call createLedger { id: #(ledgerId), fiscalYearId: #(pastFiscalYearId), restrictEncumbrance: false, restrictExpenditures: false }
+    * def v = call createFund { id: #(fundId), ledgerId: #(ledgerId) }
+    * def v = call createBudget { id: #(budgetId), allocated: 100, fundId: #(fundId), status: Active, fiscalYearId: #(pastFiscalYearId) }
```

Comment:
Can we add budget with present fiscal year id to compare that payment was created for previous FY and present fiscal year budget doesn't change

Reply:
I think you're right. Thanks